src/plugins/kibana/public/management/sections/indices/_edit.js

```diff
@@ -1,38 +1,47 @@
 import _ from 'lodash';
-import 'plugins/kibana/settings/sections/indices/_indexed_fields';
-import 'plugins/kibana/settings/sections/indices/_scripted_fields';
-import 'plugins/kibana/settings/sections/indices/_index_header';
-import PluginsKibanaSettingsSectionsIndicesRefreshKibanaIndexProvider from 'plugins/kibana/settings/sections/indices/_refresh_kibana_index';
+import 'plugins/kibana/management/sections/indices/_indexed_fields';
+import 'plugins/kibana/management/sections/indices/_scripted_fields';
+import 'plugins/kibana/management/sections/indices/_index_header';
+import RefreshKibanaIndex from 'plugins/kibana/management/sections/indices/_refresh_kibana_index';
 import UrlProvider from 'ui/url';
-import PluginsKibanaSettingsSectionsIndicesFieldTypesProvider from 'plugins/kibana/settings/sections/indices/_field_types';
+import IndicesFieldTypesProvider from 'plugins/kibana/management/sections/indices/_field_types';
 import uiRoutes from 'ui/routes';
 import uiModules from 'ui/modules';
-import editTemplate from 'plugins/kibana/settings/sections/indices/_edit.html';
+import editTemplate from 'plugins/kibana/management/sections/indices/_edit.html';
 
 uiRoutes
-.when('/settings/indices/:indexPatternId', {
+.when('/management/kibana/indices/:indexPatternId', {
   template: editTemplate,
   resolve: {
     indexPattern: function ($route, courier) {
       return courier.indexPatterns.get($route.current.params.indexPatternId)
-      .catch(courier.redirectWhenMissing('/settings/indices'));
+      .catch(courier.redirectWhenMissing('/management/data/index'));
     }
   }
 });
 
-uiModules.get('apps/settings')
-.controller('settingsIndicesEdit', function ($scope, $location, $route, config, courier, Notifier, Private, AppState, docTitle) {
+uiRoutes
+.when('/management/kibana/indices', {
+  resolve: {
+    _: function ($location, config) {
+      $location.path(`/management/kibana/indices/${config.get('defaultIndex')}`);
```

Comment:
I believe this is where it would be useful to look into something like https://docs.angularjs.org/api/ng/service/$location#replace to prevent a history record from being added so the back button is still functional.